config/index.js

```diff
@@ -75,57 +160,129 @@ module.exports = {
     'no-useless-computed-key': 'error',
+    'no-useless-concat': 'error',
     'no-useless-constructor': 'error',
     'no-useless-escape': 'error',
+    'no-useless-rename': 'error',
+    'no-useless-return': 'error',
     'no-var': 'error',
+    'no-void': 'error',
+    'no-warning-comments': 'off',
     'no-whitespace-before-property': 'error',
     'no-with': 'error',
+    'nonblock-statement-body-position': 'error',
+    'object-curly-newline': ['error', {
+      'consistent': true,
+      'multiline': true,
+    }],
     'object-curly-spacing': ['error', 'always'],
+    'object-property-newline': 'off',
+    'object-shorthand': 'error',
     'one-var': ['error', { 'initialized': 'never' }],
+    'one-var-declaration-per-line': 'off',
+    'operator-assignment': 'error',
     'operator-linebreak': ['error', 'after', { 'overrides': { '?': 'ignore', ':': 'ignore' } }],
     'padded-blocks': 'off',
+    'padding-line-between-statements': [
+      'error',
+      { blankLine: 'always', prev: 'directive', next: '*' },
+      { blankLine: 'any', prev: 'directive', next: 'directive' },
+    ],
+    'prefer-arrow-callback': 'off',
     'prefer-const': 'error',
+    'prefer-destructuring': 'error',
+    'prefer-exponentiation-operator': 'off',
+    'prefer-named-capture-group': 'off',
+    'prefer-numeric-literals': 'error',
+    'prefer-object-spread': 'error',
+    'prefer-promise-reject-errors': 'error',
+    'prefer-reflect': 'off', // Deprecated in ESLint v3.9.0
+    'prefer-regex-literals': 'error',
+    'prefer-rest-params': 'error',
+    'prefer-spread': 'error',
+    'prefer-template': 'error',
+    'quote-props': 'off',
     'quotes': ['error', 'single', { 'avoidEscape': true, 'allowTemplateLiterals': true }],
+    'radix': 'error',
+    'require-atomic-updates': 'error',
+    'require-await': 'error',
+    'require-jsdoc': 'off', // Deprecated in ESLint v5.10.0
+    'require-unicode-regexp': 'error',
+    'require-yield': 'error',
+    'rest-spread-spacing': 'error',
     'semi': ['error', 'never'],
     'semi-spacing': ['error', { 'before': false, 'after': true }],
+    'semi-style': 'error',
+    'sort-imports': 'off',
+    'sort-keys': 'off',
+    'sort-vars': 'off',
     'space-before-blocks': ['error', 'always'],
     'space-before-function-paren': ['error', 'always'],
     'space-in-parens': ['error', 'never'],
     'space-infix-ops': 'error',
     'space-unary-ops': ['error', { 'words': true, 'nonwords': false }],
-    'spaced-comment': ['error', 'always', { 'markers': ['global', 'globals', 'eslint', 'eslint-disable', '*package', '!', ','], 'exceptions': ['=', '-'] } ],
+    'spaced-comment': ['error', 'always', { 'markers': ['global', 'globals', 'eslint', 'eslint-disable', '*package', '!', ','], 'exceptions': ['=', '-'] }],
     'strict': 'off',
+    'switch-colon-spacing': 'error',
+    'symbol-description': 'error',
     'template-curly-spacing': ['error', 'never'],
+    'template-tag-spacing': ['error', 'never'],
```

Comment:
Nit: the `never` setting is the default
```suggestion
    'template-tag-spacing': 'error',
```